script.js

```diff
@@ -0,0 +1,164 @@
+// 1
+
+const teamRegistration = (
+  teamName,
+  driverName,
+  yearOfbirth,
+  carBrand,
+  sponsor,
+  allowedTorally
+) => {
+  teamName = prompt("Enter name of your team");
+  driverName = prompt("Enter your name");
+  yearOfbirth = +prompt("Enter your year of birth");
+  carBrand = prompt("Enter your car brand");
+  sponsor = prompt("Enter name of your sponsor");
+
+  if (
+    teamName === null ||
+    driverName === null ||
+    yearOfbirth === null ||
+    carBrand === null ||
+    sponsor === null
+  ) {
+    allowedTorally = false;
+  } else if (
+    teamName === "" ||
```

Comment:
As an improvement validations for empty strings and null can be moved to a helper function